lib/salesforce_service.rb

```diff
@@ -0,0 +1,84 @@
+require 'restforce'
+
+class SalesforceService
+  class << self
+    def add_artwork(submission_id)
+      return unless api_enabled?
+
+      submission = Submission.find(submission_id)
+      
+      sf_contact_id = find_contact_id(submission.user_id) || api.create!('Contact', map_submission_to_salesforce_contact(submission))
+      sf_artist_id = api.select('Artist__c', submission.artist_id, ['Id'], 'Gravity_Artist_ID__c').Id
```

Comment:
can we assume the artist will always be found?

Reply:
I would say yes, assume the artist will be found otherwise raise. We are already using the bang methods here so it seems like raising is the way we've chosen to go. Also there isn't really a default artist we can use.

Reply:
I agree! For reference, the default behaviour of the `select` is to raise (for example, we're catching that in the `find_contact_id` method) so it should be fine as is